pingone-netcore-sdk/PingOne.Core/Management/Services/ManagementApiClient.cs

```diff
@@ -88,13 +88,16 @@ public async Task<string> GetPasswordRegexPattern()
 
                 passwordPattern.Append(")");
 
-                passwordPattern.Append("(?!.*(.)\\1{");
-                passwordPattern.Append(defaultPolicy.MaxRepeatedCharacters);
-                passwordPattern.Append(",}).{");
-                passwordPattern.Append($"{defaultPolicy.Length.Min},{defaultPolicy.Length.Max}");
+                if (defaultPolicy.MaxRepeatedCharacters > 0)
+                {
+                    passwordPattern.Append("(?!.*(.)\\1{");
+                    passwordPattern.Append(defaultPolicy.MaxRepeatedCharacters);
+                    passwordPattern.Append(",})");
+                }
+                passwordPattern.Append($".{{{defaultPolicy.Length.Min},{defaultPolicy.Length.Max}}}");
```

Comment:
Nice!
But I guess there are too much `{{{`. And should be
```
passwordPattern.Append(".{");
passwordPattern.Append($".{defaultPolicy.Length.Min},{defaultPolicy.Length.Max}");
passwordPattern.Append("}$");
```

Reply:
Yes, that is more readable.
I will make the change.